website_portal_medical_prescription_order_line/controllers/main.py

```diff
@@ -28,6 +28,21 @@ def my_medical(self, **kw):
             ('patient_id.parent_id', 'child_of', [partner_id.id]),
         ])
 
+        url = str(request.httprequest)
```

Comment:
The issue is that I would select multiple patients, but when I requested the "patients" parameter, it would only ever give me the first patient selected and not a list of all the patients I had selected. So here, I am getting the URL and getting each patient's id from the parameters that show up there.